src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -5317,7 +5314,11 @@ private void CheckPossibleNullReceiver(BoundExpression receiverOpt, bool checkNu
                     {
                         return;
                     }
-                    ReportSafetyDiagnostic(isValueType ? ErrorCode.WRN_NullableValueTypeMayBeNull : ErrorCode.WRN_NullReferenceReceiver, syntaxOpt ?? receiverOpt.Syntax);
+
+                    if (!receiverOpt.IsSuppressed)
```

Comment:
>if (!receiverOpt.IsSuppressed) [](start = 20, length = 30)

Is this check necessary? Won't `_resultType` have been set appropriately by the caller?

Reply:
Thanks. Indeed. I think I added this before I fixed the result type from ` VisitExpressionWithoutStackGuard` and should have revisited this afterwards.

---
In reply to: [251524384](https://github.com/dotnet/roslyn/pull/32850#discussion_r251524384) [](ancestors = 251524384)